briskedu/brisk-learning
# Simplify front-end states

Summary:
Simplify front-end states in preparation for adding a three dot options menu.

Test Plan:
1. Confirm login with errors shows permission prompt
2. Confirm error shows up and can be collapsed
3. Confirm num edits can be expanded and collapsed
4. Confirm checking for AI shows menu and can be expanded and collapsed
5. Confirm collapsing when toggling between classroom assignments works
